esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -186,8 +186,11 @@ void CWsESDLConfigEx::init(IPropertyTree *cfg, const char *process, const char *
     if(servicecfg == NULL)
         throw MakeStringException(-1, "config not found for service %s/%s",process, service);
 
-    m_isDetachedFromDali = false;
     m_esdlStore.setown(createEsdlCentralStore());
+    if(nullptr == m_esdlStore)
+        m_isDetachedFromDali = true;
+    else
+        m_isDetachedFromDali = false;
```

Comment:
When the CWsESDLConfigEx::attachServiceToDali() is called, the m_isDetachedFromDali will be set to false. Should the m_esdlStore be set and checked there?